config/__init__.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'''This module handles application configuration and secrets.'''
+
+import os
+from datetime import datetime
+from collections import namedtuple
+
+from environment import *
+from secrets import *
```

Comment:
don't like global import like these, but don't have a better solution. So let's fly with it. ;)